eslint.config.js

```diff
@@ -1,29 +1,20 @@
-const cds = require('@sap/eslint-plugin-cds')
+const eslintCds = require('@sap/eslint-plugin-cds')
+const eslintJs = require('@eslint/js')
 const globals = require('globals')
-const js = require('@eslint/js')
 
 module.exports = [
-  cds.configs.recommended,
-  js.configs.recommended,
+  eslintJs.configs.recommended,
+  eslintCds.configs.recommended,
   {
     languageOptions: {
       globals: {
-        es2022: true,
+        sap: true,
+        ...globals.es2022,
         ...globals.browser,
         ...globals.node,
         ...globals.jest,
         ...globals.mocha,
-        cds: true,
-        sap: true,
-        CDL: true,
-        CQL: true,
-        CREATE: true,
-        DELETE: true,
-        DROP: true,
-        INSERT: true,
-        SELECT: true,
-        UPDATE: true,
-        UPSERT: true
+        ...eslintCds.configs.recommended.languageOptions.globals
```

Comment:
in addition to the constants replaced above, `@sap/eslint-plugin-cds` also exports `CXL: true` (_lib/constants.js_) which was not explicitly part of the exposed `globals`. Is this done on purpose?

Reply:
it's part of the overall cds config and maybe missied out